pywebio/platform/fastapi.py

```diff
@@ -151,8 +151,41 @@ def start_server(applications, port=0, host='',
 
     .. versionadded:: 1.3
     """
-    kwargs = locals()
+    app = build_starlette_app(allowed_origins, applications, cdn, check_origin, debug, static_dir)
```

Comment:
It's better to use keyword arguments:
```suggestion
    app = build_starlette_app(applications, cdn=cdn, static_dir=static_dir, debug=debug, allowed_origins=allowed_origins, check_origin=check_origin)
```